### DISCUSSION TITLE
How to use volume.check.disk on a read-only volume

### DISCUSSION THREAD

**Discussion by amir-latifi** [seaweedfs/seaweedfs#5926] [Q&A]
Hi, 
I have some volumes with replica placement of 001. They have inconsistency. It means they have entries in replica #1 which are not in replica #2 and entries in replica #2 which are not in replica #1. 
I know i can use volume.check.disk to fix this issue, but it does not work on read-only volumes. How can I fix these volumes? Is there any command or manual way for it?

**Comment by chrislusf** [comment]
usually if the volumes are inconsistent, the write operations would fail and the entry would not be used. 

you can use `see_dat` under https://github.com/seaweedfs/seaweedfs/tree/master/unmaintained/see_dat to list out and compare the entries.

how to fix the data is tricky. you can share the volume data to see what happened?
